test/View/Helper/FormDateSelectTest.php

```diff
@@ -115,4 +115,32 @@ public function testGetElements(): void
 
         $this->assertCount(31, $elements[0]->getValueOptions());
     }
+
+    /**
+     * @group issue-160
+     */
+    public function testUnescapedCharacter(): void
+    {
+        $element  = new DateSelect('foo');
+        $dateType = IntlDateFormatter::SHORT;
+        $locale   = 'es_CL';
+
+        $this->helper->__invoke(
+            $element,
+            $dateType,
+            $locale
+        );
+
+        $this->assertSame($locale, $this->helper->getLocale());
+        $this->assertSame($dateType, $this->helper->getDateType());
+
+        $elements = $element->getElements();
+        $this->assertCount(3, $elements);
+
+        foreach ($elements as $subElement) {
+            $this->assertInstanceOf(Select::class, $subElement);
+        }
+
+        $this->assertCount(31, $elements[0]->getValueOptions());
```

Comment:
I feel like this test isn't really readable: what are we testing here, specifically? 🤔

Reply:
fixed